### DISCUSSION TITLE
Entity Ding in Home Assistant (Ring Intercom)

### DISCUSSION THREAD

**Discussion by Thundervf** [tsightler/ring-mqtt#683] [Support Q&A]
Hi. About 3 months ago I installed mqtt-ring on my home Assistant to integrate my Ring Intercom. At that time the "Ding" entity was working correctly, it changed the state from off to on when someone pressed the button. Since a month ago, said entity is not changing the state, it stays in "off". Any solution to get it working again?

**Comment by tsightler** [comment]
Please make sure you are on the latest version and follow the steps in the release notes for v5.3.0:

https://github.com/tsightler/ring-mqtt/blob/main/docs/CHANGELOG.md#v530

**Reply by Thundervf** [reply]
Thank u so much :)
